helm/charts/determined/templates/master-deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           {{ $tag = .Values.detVersion }}
         {{- end -}}
         image: {{ .Values.imageRegistry }}/{{ $image }}:{{ $tag }}
-        imagePullPolicy: "Always"
+        imagePullPolicy: "Never"
```

Comment:
ditto

Reply:
yeah, i need this for my testing flow (this is how you use 0.31.1-dev0 images locally with a helm install). i won't forget to remove it.